src/main/java/org/mastodon/mamut/clustering/ui/DendrogramPanel.java

```diff
@@ -207,6 +229,25 @@ private void paintLineLegend( final Graphics2D g2, final String text, final int
 		}
 	}
 
+	void export( final File file, final String formatName, int screenResolution )
```

Comment:
I guess the usecase for this code, is that a user wants to export a dendrogram to use it in a publication. This is a very relevant and import feature. I would suggest two changes:
1. Save the dendrogram as vectory graphic. The class SVGGraphics2D in the library JFreeSVG can be used for this, and it is part of every Fiji installation.
2. Let the user specify the width an height of the exported graphic. I know that would require a dialog and it's extra work. But users most likely will combine more than one dendrogram in a figure and they need to be of a well defined size for this.

Reply:
> I guess the usecase for this code, is that a user wants to export a dendrogram to use it in a publication. This is a very relevant and import feature. I would suggest two changes:
> 
>     1. Save the dendrogram as vectory graphic. The class SVGGraphics2D in the library JFreeSVG can be used for this, and it is part of every Fiji installation.

Good suggestion. I added the option of an SVG export.

>     2. Let the user specify the width an height of the exported graphic. I know that would require a dialog and it's extra work. But users most likely will combine more than one dendrogram in a figure and they need to be of a well defined size for this.

At the moment it is possible to export with the default size. Then multiple dendrograms can be exported with the same size to either png or svg. I suggest to wait for the user's feedback, if it is required to specify sizes different from the default size for exports to avoid potentially unnecessary effort.